lib/Command/SetConfig.php

```diff
@@ -60,6 +60,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		$availableConfigs = $helper->getServerConfigurationPrefixes();
 		$configID = $input->getArgument('configID');
 		if(!in_array($configID, $availableConfigs)) {
+			var_dump($availableConfigs);
```

Comment:
bonus debugging code

Reply:
will have to be cleaned up before merging, would like to keep it while [WIP]

Reply:
Debug line is now gone 👍